src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,41 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When did World Wide Web Consortium first drafted standards for developing accessible websites?",
+    Answer: "1999",
+    Distractor1: "2001",
+    Distractor2: "1990",
+    Distractor3:
+      "1989",
```

Comment:
this could be on the same line

```suggestion
    Distractor3: "1989",
```